docs/pages/admin-guides/management/operations/ca-rotation.mdx

```diff
@@ -116,10 +125,20 @@ Service instances have completed the transition to target phase before
 proceeding to the next phase. We will explain the phases in [Step
 2](#step-24-start-a-manual-rotation).
 
+<Admonition type="note">
+Any OpenSSH hosts must be issued new host certificates during the
+`update_servers` phase of the `host` CA rotation.
+</Admonition>
+
+<Admonition type="note">
 If you are joining Teleport processes to a cluster via the Teleport Auth
-Service, each Teleport process will need a CA pin to trust the Auth Service.
-The CA pin will change after each `host` CA rotation. Make sure you use the
-*new* CA pin when adding Teleport services after `host` CA rotation.
+Service, each Teleport process needs a CA pin to trust the Auth Service.
+Teleport Agents connecting to a Proxy Service address never need a CA pin, but
+new Proxy Services should always use a CA pin when joining the cluster.
+During the CA rotation, `tctl status` will report that there are 2 CA pins.
+The CA pin configuration can accept a list including both pins.
+After the rotation is complete, only the new CA pin will be reported.
```

Comment:
I would work these Admonitions into the body text. Successive Admonitions look cluttered and difficult to read.

Reply:
made both of these normal paragraphs